x-pack/plugins/alerting/server/lib/wrap_scoped_cluster_client.ts

```diff
@@ -22,13 +22,9 @@ import type {
 } from '@elastic/elasticsearch/lib/api/typesWithBodyKey';
 import { IScopedClusterClient, ElasticsearchClient, Logger } from '@kbn/core/server';
 import { Rule } from '../types';
-import { RuleRunMetrics } from './rule_run_metrics_store';
+import { SearchMetrics } from './types';
 
 type RuleInfo = Pick<Rule, 'name' | 'alertTypeId' | 'id'> & { spaceId: string };
```

Comment:
Should we remove this and use the `RuleInfo` as defined in `x-pack/plugins/alerting/server/lib/types.ts` instead?

Reply:
Done.